docs/src/booklet/2-qubo.md

```diff
@@ -0,0 +1,34 @@
+# QUBO
+
+## Definition
+QUBO stands for Quadratic Unconstrained Binary Optimization. 
+
+So every QUBO model has the following characteristics:
+- Its objective function is at most quadratic
+- There are no constraints
+- All variables are binary (0 or 1)
+
+We can represent a QUBO as follows.
+
+```math
+\begin{array}{rl}
+   \min        & \mathbf{x}' Q\,\mathbf{x} \\
+   \text{s.t.} & \mathbf{x} \in \mathbb{B}^{n}
+\end{array}
+```
+
+## OK, but why QUBO?
+
+
+QUBOs are suited for representing non-convex global optimization problems.
+With that said, the significant advances in computing systems and algorithms specialized for sampling QUBOs have contributed to their popularity.
```

Comment:
Answering why QUBO requires to say:
- Equivalence to MAXCUT (from combinatorial optimization)
- Equivalence to Ising model from Physics